src/backend/distributed/commands/create_citus_local_table.c

```diff
@@ -561,13 +561,14 @@ RenameShardRelationIndexes(Oid shardRelationId, uint64 shardId)
  * the index with indexName.
  */
 static char *
-GetRenameShardIndexCommand(char *indexName, uint64 shardId)
+GetRenameShardIndexCommand(Oid indexOid, uint64 shardId)
 {
+	char *indexName = get_rel_name(indexOid);
 	char *shardIndexName = pstrdup(indexName);
 	AppendShardIdToName(&shardIndexName, shardId);
 	const char *quotedShardIndexName = quote_identifier(shardIndexName);
```

Comment:
I wonder if we should qualify schema name for new name of the index

Reply:
I suppose the syntax doesn't include a schema name for the new name